tools/manifest/download.py

```diff
@@ -153,11 +145,8 @@ def download_manifest(
                 logger.warning("Failed to decompress downloaded file")
                 continue
         elif url.endswith(".gz"):
-            fileobj = io.BytesIO(resp.read())
             try:
-                with gzip.GzipFile(fileobj=fileobj) as gzf:
-                    data = gzf.read()
-                    decompressed = data
+                decompressed = gzip.decompress(resp.read())
```

Comment:
This is simplification made possible by `gzip.decompress` being added in Python 3.2.